src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,65 @@
  */
 
 public class Plane {
+    private int id;
+    private int weight;
+    private int speed;
+    private String model;
+    private String color;
 
+    private Plane(PlaneBuilder planeBuilder) {
+        this.id = planeBuilder.id;
```

Comment:
```suggestion
        id = planeBuilder.id;
```
аналоігчно з рештою)